src/routes/safe/components/Apps/components/AppFrame.tsx

```diff
@@ -264,9 +264,18 @@ const AppFrame = ({ appUrl }: Props): ReactElement => {
     })
 
     communicator?.on(Methods.signMessage, async (msg) => {
+      // const { message } = msg.data.params as SignMessageParams
+      const message =
+        '{"types":{"Person":[{"name":"name","type":"string"},{"name":"wallet","type":"address"}],"Mail":[{"name":"from","type":"Person"},{"name":"to","type":"Person"},{"name":"contents","type":"string"}]},"primaryType":"Mail","domain":{"name":"Ether Mail","version":"1","chainId":1,"verifyingContract":"0xCcCCccccCCCCcCCCCCCcCcCccCcCCCcCcccccccC"},"message":{"from":{"name":"Cow","wallet":"0xCD2a3d9F938E13CD947Ec05AbC7FE734Df8DD826"},"to":{"name":"Bob","wallet":"0xbBbBBBBbbBBBbbbBbbBbbbbBBbBbbbbBbBbbBBbB"},"contents":"Hello, Bob!"}}'
+
+      openSignMessageModal(message, msg.data.id, Methods.signTypedMessage)
+      // openSignMessageModal(message, msg.data.id, Methods.signMessage)
```

Comment:
Looks like the PR isn't ready for review. Please convert it to draft and let us know when we can look at it.

Reply:
Sorry, the code is updated and ready for review. This is intended for testing but I forgot to remove it... Really sorry.

Reply:
All good, thank you!